src/mame/casio/fp1100.cpp

```diff
@@ -714,7 +714,8 @@ ROM_START( fp1100 )
 	ROM_LOAD( "sub3.rom", 0x2000, 0xf80, BAD_DUMP CRC(fb2b577a) SHA1(a9ae6b03e06ea2f5db30dfd51ebf5aede01d9672))
 ROM_END
 
-// FP1000 has less memory than FP1100, using the same PCB, but with unpopulated sockets from RAM9 to RAM24 (only RAM1 to RAM8 are populated).
+/* FP1000 has less memory than FP1100, using the same PCB, but with unpopulated sockets from RAM9 to RAM24 (only RAM1 to RAM8 are populated).
+   The dumped PCB has the uPD7801G socket labeled as "uPD7801G-101", but the chip seems like a newer revision, silkscreened as "D7108G-118". */
```

Comment:
Do you have any evidence that systems ever shipped with a chip with mask code 101?  All the photos I’ve seen of FP1100 and FP1000 boards show a µPD7801G with mask code 118 while the PCB has “-101” on the silk screen.  The µPD7801G is sometimes socketed and sometimes soldered.